rest-api/tools/auth_setup.sh

```diff
@@ -88,10 +88,24 @@ function get_instance_connection_name {
 }
 
 function get_db_password {
+  user=$1
   echo "Getting database password..."
   tools/install_config.sh --key db_config --instance $INSTANCE \
       --creds_file ${CREDS_FILE} --config_output $TMP_DB_INFO_FILE
-  PASSWORD=`grep db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  if [ "${user}" == "${RDR_DB_USER}" ]
+  then
+      PASSWORD=`grep db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  elif [ "${user}" == "${READONLY_DB_USER}" ]
+  then
+      PASSWORD=`grep read_only_db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  elif [ "${user}" == "${ROOT_DB_USER}" ]
+  then
+      PASSWORD=`grep root_db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  echo "This is a test --------------------------------------------------"
+  echo "user is $user"
+  echo "PASSWORD IS $PASSWORD"
+  echo " END TEST ======================================================="
+  fi
 }
```

Comment:
We may want to add a check here to make sure PASSWORD is set to something after this runs, and print an error and exit if it isn't. (Particularly since existing db_config entries won't have the new root and readonly password fields.)

Reply:
done